src/filesystemutilities.cpp

```diff
@@ -12,7 +12,7 @@ bool fixDirectoryName(QString& name)
   while (temp.endsWith('.'))
     temp.chop(1);
 
-  temp.replace(QRegularExpression("[<>:\"/\\|?*]"), "");
+  temp.replace(QRegularExpression("[<>:\"/\\\\|?*]"), "");
```

Comment:
Regex101 tells me it should be `[<>:\"\/\\|?*]` 
The main issue in both version here in the diff is that they seem to have an unescaped `/`.

Don't know if we need to account for c++ string escaping on top of that, so might be worth using a Raw string literal:
https://en.cppreference.com/w/cpp/language/string_literal

Reply:
Hello, the backslash change is based on an example from a QRegularExpression documentation page https://doc.qt.io/qt-6/qregularexpression.html It shows ```Note that due to C++ literal strings rules, you must escape all backslashes inside the pattern string with another backslash: QRegularExpression re2("\\\\");```. I believe ```QRegularExpression("[<>:\"/\\\\|?*]")``` and ```QRegularExpression(R"([<>:"/\\|?*])")``` are equivalent if you would like it to be a raw string literal.

Regarding the ```/```, it seems to be working correctly without being escaped. I did not see anything in the documentation specifically about that character though.

Reply:
You are right, it seems some languages need `\` to be escaped, but doesn't seem c++ needs it.
Yeah I would go with the Raw version just so we don't mix in c++ escaping with regex escaping, should make the it simpler to understand.

Reply:
Sounds good, I've changed it to use the Raw version.

Reply:
Perfect, thanks!